sk/music-optimized/Menu.asm

```diff
@@ -635,42 +639,42 @@ Snd_Menu_Loop06:
 	dc.b	$36
 	smpsCall            Snd_Menu_Call01
 	smpsCall            Snd_Menu_Call01
-	smpsFMAlterVol      $FD, $D3
+	;smpsFMAlterVol      $FD, nMaxPSG1
 	dc.b	$06
-	smpsFMAlterVol      $03, $D3
+	;smpsFMAlterVol      $03, nMaxPSG1
 	dc.b	$03, nMaxPSG1
 
 Snd_Menu_Loop07:
 	dc.b	nMaxPSG1, $06
 	smpsLoop            $00, $0D, Snd_Menu_Loop07
 	dc.b	nMaxPSG1, $32, nMaxPSG1, $34, nMaxPSG1, $06, nMaxPSG1, $03, nMaxPSG1, nMaxPSG1, $06
-	smpsFMAlterVol      $FD, $D3
-	smpsFMAlterVol      $03, $D3
+	;smpsFMAlterVol      $FD, nMaxPSG1
+	;smpsFMAlterVol      $03, nMaxPSG1
 	dc.b	nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1
 
 Snd_Menu_Loop08:
 	dc.b	nMaxPSG1, $06, nMaxPSG1, $03, nMaxPSG1, nMaxPSG1, $06
-	smpsFMAlterVol      $FD, $D3
-	smpsFMAlterVol      $03, $D3
+	;smpsFMAlterVol      $FD, nMaxPSG1
+	;smpsFMAlterVol      $03, nMaxPSG1
 	dc.b	nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1
 	smpsLoop            $00, $06, Snd_Menu_Loop08
 	dc.b	nMaxPSG1, $06, nMaxPSG1, $03, nMaxPSG1, nMaxPSG1, $06
-	smpsFMAlterVol      $FD, $D3
-	smpsFMAlterVol      $03, $D3
+	;smpsFMAlterVol      $FD, nMaxPSG1
+	;smpsFMAlterVol      $03, nMaxPSG1
 	dc.b	nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1
 	dc.b	nMaxPSG1, $03, nMaxPSG1, nMaxPSG1, $06
-	smpsFMAlterVol      $FD, $D3
-	smpsFMAlterVol      $03, $D3
+	;smpsFMAlterVol      $FD, nMaxPSG1
+	;smpsFMAlterVol      $03, nMaxPSG1
 	dc.b	nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1, nMaxPSG1
 	dc.b	nMaxPSG1, $03, nMaxPSG1, nMaxPSG1, $06
-	smpsFMAlterVol      $FD, $D3
+	;smpsFMAlterVol      $FD, nMaxPSG1
```

Comment:
Hm, except for this one. This is unbalanced, and will cause the music to fade to silence over time.